scholarx/2023/index.html

```diff
@@ -0,0 +1,698 @@
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" rel="stylesheet">
+    <!-- Argon CSS -->
+    <link type="text/css" href="/assets/css/argon.css?v=1.0.1" rel="stylesheet">
+    <!-- global CSS -->
+    <link type="text/css" href="/assets/css/global-custom.css" rel="stylesheet">
+    <!-- scholarx CSS -->
+    <link type="text/css" href="styles.css" rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+
+<body>
+<header class="header-global">
+    <!--nav bar goes here-->
+    <div id="navbar"></div>
+</header>
+
+<section class="section pb-0 bg-gradient-white mt-5">
+    <div class="container">
+        <div class="row row-grid align-items-center justify-content-center">
+            <div class="col-md-6">
+                <div class="d-flex px-3">
+                    <div class="pl-4">
+                        <img src="/assets/img/brand/scholarx.png" class="img-fluid align-self-center mb-5 w-50">
+                        <p>ScholarX is an exclusive 6-month program aimed at providing
+                            mentoring support to a selected pool of high Potential undergraduate students
+                            based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
+                            the world’s top universities or Fortune 500 companies. It’s our free premium
+                            mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
+                            of knowledge and expertise sharing without the limitation of geographical borders.</p>
+                    </div>
+                    
+                </div>
+            </div>
+            <div class="col-md-6">
+                <img src="/assets/img/scholarx/new-findmentor.png" class="img-fluid align-self-center w-100">
+            </div>
+        </div>
+    </div>
+</section>
+
+<!-- Join Mentor and Mentee waiting lists -->
+<section class="section section-secondary">
+  <div id="mentor-waiting-list" class="container-fluid">
+    <div class="row justify-content-center">
+      <div class="col-9 text-center">
+        <h1 class="mb-0 text-onelive">Become a mentor for the </h1>
+        <p class="display-3 text-onelive font-weight-bolder">2023 program</p>
+      </div>
+    </div>
+    <div class="row justify-content-center">
+      <div class="col-9 text-center">
+        <a href="https://docs.google.com/forms/u/3/d/e/1FAIpQLSeI56qNCzFOnq2vM_PXMfD6iys1Zw0sPQDgsd1ySIHDXLpSpA/viewform?usp=send_form" target="_blank" class="btn btn-default">Join Mentor Waiting List</a>
+      </div>
+    </div>
+  </div>
+</section>
+<section class="section section-secondary">
+  <div id="mentee-waiting-list" class="container-fluid">
+    <div class="row justify-content-center">
+      <div class="col-9 text-center">
+        <h1 class="mb-0 text-onelive">Sign-up here to get a notification </h1>
```

Comment:
h1 seems too big maybe h2 
```suggestion
        <h2 class="mb-0 text-onelive">Sign-up here to get a notification </h2>
```